### PR TITLE
Feature/wireframe mesh

### PR DESCRIPTION
**Pull request by r-englund**
Move code from https://github.com/inviwo/modules/pull/4 (the OpenMesh PR)

### CODE REVIEW

File: modules/base/src/processors/trianglestowireframe.cpp [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+#include <modules/base/algorithm/meshutils.h>
+
+namespace inviwo {
+
+// The Class Identifier has to be globally unique. Use a reverse DNS naming scheme
+const ProcessorInfo TrianglesToWireframe::processorInfo_{
+    "org.inviwo.TrianglesToWireframe",  // Class identifier
+    "Triangles To Wireframe",           // Display name
+    "Mesh Operation",                   // Category
+    CodeState::Experimental,            // Code state
```
Comment (ResearchDaniel) [inline modules/base/src/processors/trianglestowireframe.cpp]: Stable?

**Review by ResearchDaniel — APPROVED** [review]

File: modules/base/include/modules/base/processors/trianglestowireframe.h [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+ *
+ * ### Inports
+ *   * __mesh__ <description>.
+ *
+ * ### Outports
+ *   * __wireframe__ <description>.
+ */
+
+/**
+ * \brief VERY_BRIEFLY_DESCRIBE_THE_PROCESSOR
```
Comment (martinfalk) [inline modules/base/include/modules/base/processors/trianglestowireframe.h]: remove

**Review by martinfalk — APPROVED** [review]

File: modules/base/include/modules/base/processors/trianglestowireframe.h (hunk 2 above)
Comment (r-englund) [reply modules/base/include/modules/base/processors/trianglestowireframe.h]: I did that 9 minutes before your comment ;)

File: modules/base/include/modules/base/algorithm/meshutils.h [hunk 3]
```diff
@@ -103,6 +103,86 @@ IVW_MODULE_BASE_API std::shared_ptr<ColoredMesh> cameraFrustum(
...
+        uint32_t a = static_cast<uint32_t>(ram.front());
+        for (size_t i = 1; i < ram.size(); i++) {
+            callback(a, ram[i], ram[i + 1]);
+        }
+    }
+
+    else {
+        std::ostringstream errMsg;
+        errMsg << "ConnectivityType " << info.ct << " not supported";
+        throw inviwo::Exception(errMsg.str(), IvwContextCustom("meshutil::forEachTriangle"));
```
Comment (petersteneteg) [inline modules/base/include/modules/base/algorithm/meshutils.h]: Use IVW_CONTEXT_CUSTOM

**Pull request merged**